wordpress-mobile/gutenberg-mobile
# Add nvmrc file to specify node version

This PR adds an `.nvmrc` file to specify the project's preferred node version. I've copied over Gutenberg's `.nvmrc` file which specifies the node LTS version. This PR doesn't shouldn't change anything with regard to which node version we're using, it's specifying it here in this repo instead of relying on Gutenberg repo's `.nvmrc`.

This _might_ also help an issue where our `package-lock.json` file gets changes after running `npm i`. This issue could be related to unintentional use of different node versions.

**I don't have much knowledge of how buildkite is configured here, so apart from verifying that the buildkite checks are green, I haven't done much else to check this change's impact on buildkite.**

### To test
1. Verify buildkite CI check passes
2. Verify CircleCI check passes
3. Follow development environment setup instructions and verify the still achieve their goal of getting the contributor to install the right node version

PR submission checklist:
